### PR TITLE
chore: [Hub Bot] Refresh metadata 2024-10-07

### PR DESCRIPTION
**Pull request by MeltyBot**
Updates Plugin Definitions

### CODE REVIEW

File: _data/meltano/loaders/target-snowflake/meltanolabs.yml [hunk 1]
```diff
@@ -93,12 +93,7 @@ settings:
   label: Private Key Passphrase
   name: private_key_passphrase
   sensitive: true
-- description: Private key contents. For KeyPair authentication either `private_key` or `private_key_path` must be provided.
```
Comment (pnadolny13) [inline _data/meltano/loaders/target-snowflake/meltanolabs.yml:96]: 🤔

Comment (edgarrmondragon) [reply _data/meltano/loaders/target-snowflake/meltanolabs.yml:96]: Yeah, the `pip_url` does point to the PyPI distribution, and the latest there is `0.11.0` which does contain the commit that added the setting so I'm baffled.

File: _data/meltano/extractors/tap-mailchimp/acarter24.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ settings:
   sensitive: true
 - description: Your Mailchimp DC
   kind: string
-  label: Data Center
+  label: Dc
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-mailchimp/acarter24.yml:31]:
```suggestion
  label: Data Center
```

File: _data/meltano/extractors/tap-totango/edsoncezar16.yml [hunk 3]
```diff
@@ -76,9 +76,9 @@ settings:
   label: API URL
   name: api_url
   options:
-  - label: US Endpoint
+  - label: Https://API Totango Com
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-totango/edsoncezar16.yml:79]:
```suggestion
  - label: US Endpoint
```

File: _data/meltano/extractors/tap-totango/edsoncezar16.yml [hunk 4]
```diff
@@ -76,9 +76,9 @@ settings:
...
     value: https://api.totango.com
-  - label: EU Endpoint
+  - label: 'Https://API Eu1 Totango Com '
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-totango/edsoncezar16.yml:81]:
```suggestion
  - label: EU Endpoint
```

File: _data/meltano/loaders/target-redshift/ticketswap.yml [hunk 5]
```diff
@@ -48,7 +48,7 @@ settings:
   sensitive: true
 - description: Database name. Note if sqlalchemy_url is set this will be ignored.
   kind: string
-  label: Database name
+  label: Databasename
```
Comment (edgarrmondragon) [inline _data/meltano/loaders/target-redshift/ticketswap.yml:51]:
```suggestion
  label: Database name
```

File: _data/meltano/loaders/target-snowflake/meltanolabs.yml [hunk 6]
```diff
@@ -93,12 +93,7 @@ settings:
...
-  kind: password
-  label: Private Key
-  name: private_key
-  sensitive: true
-- description: Path to file containing private key. For KeyPair authentication either `private_key` or `private_key_path` must be provided.
+- description: Path to file containing private key.
```
Comment (edgarrmondragon) [inline _data/meltano/loaders/target-snowflake/meltanolabs.yml:96]:
```suggestion
- description: Private key contents. For KeyPair authentication either `private_key` or `private_key_path` must be provided.
  kind: password
  label: Private Key
  name: private_key
  sensitive: true
- description: Path to file containing private key. For KeyPair authentication either `private_key` or `private_key_path` must be provided.
```

**Review by edgarrmondragon — APPROVED** [review]

File: _data/meltano/loaders/target-snowflake/meltanolabs.yml (hunk 1 above)
Comment (edgarrmondragon) [reply _data/meltano/loaders/target-snowflake/meltanolabs.yml]: I'll make a post-release and see that fixes this issue

**Pull request merged**